### PR TITLE
added more emojis

### PR DESCRIPTION
**Pull request by fedescarpa**

### CODE REVIEW

**Review by flbulgarelli — CHANGES_REQUESTED** [review]
Looks good, but please notice that we are listing emojis that are not present in bibliotheca. E.g: 

* `thinking_face` is not here https://github.com/mumuki/mumuki-bibliotheca/tree/master/src/assets/emojis. :thinking: 
* Neither is `woman_woman_boy_boy` :family_woman_woman_boy_boy: .

File: lib/mumukit/content_type/md_emoji.rb [hunk 1]
```diff
@@ -20,102 +20,114 @@
...
+    jeans joy joy_cat joystick jp kaaba key keyboard keycap_star keycap_ten kimono kiss kissing kissing_cat kissing_closed_eyes kissing_face
+    kissing_heart kissing_smiling_eyes knife knife_fork_plate koala koko kr label lantern large_blue_circle large_blue_diamond large_orange_diamond
+    last_quarter_moon last_quarter_moon_with_face latin_cross laughing leaves ledger left_luggage left_right_arrow left_speech_bubble
+    leftwards_arrow_with_hook lemon leo leopard level_slider libra light_rail lightning lightning_cloud link linked_paperclips lion_face
+    lips lipstick lock lock_with_ink_pen lollipop loop loud_sound loudspeaker love_hotel love_letter low_brightness lower_left_ballpoint_pen
+    lower_left_crayon lower_left_fountain_pen lower_left_paintbrush m mag mag_right mahjong mailbox mailbox_closed mailbox_with_mail
+    mailbox_with_no_mail man man-heart-man man-kiss-man man-man-boy man-man-boy-boy man-man-girl man-man-girl-boy man-man-girl-girl
+    man-woman-boy man-woman-boy-boy man-woman-girl man-woman-girl-boy man-woman-girl-girl man_in_business_suit_levitating man_with_gua_pi_mao
```
Comment (flbulgarelli) [inline lib/mumukit/content_type/md_emoji.rb:89]: I see some inconsistencies in naming: some emojis are `snake_case` , while others are `dash-case`. 

All them should be `snake_case`.

File: lib/mumukit/content_type/md_emoji.rb [hunk 2]
```diff
@@ -20,102 +20,114 @@
...
+    small_orange_diamond small_red_triangle small_red_triangle_down smile smile_cat smiley smiley_cat smiling_imp smirk smirk_cat smoking snail
+    snake snow_capped_mountain snow_cloud snowboarder snowflake snowman sob soccer soon sos sound space_invader spades spaghetti sparkle sparkler
+    sparkles sparkling_heart speak_no_evil speaker speaking_head_in_silhouette speech_balloon speedboat spider spider_web spiral_calendar_pad
+    spiral_note_pad spock-hand sports_medal squirrel stadium star star2 star_and_crescent star_of_david stars station statue_of_liberty
+    steam_locomotive stew stopwatch straight_ruler strawberry stuck_out_tongue stuck_out_tongue_closed_eyes stuck_out_tongue_winking_eye
+    studio_microphone sun_behind_cloud sun_behind_rain_cloud sun_small_cloud sun_with_face sunflower sunglasses sunny sunrise sunrise_over_mountains
+    surfer sushi suspect suspension_railway sweat sweat_drops sweat_smile sweet_potato swimmer symbols synagogue syringe table_tennis_paddle_and_ball
+    taco tada tanabata_tree tangerine taurus taxi tea telephone telephone_receiver telescope tennis tent the_horns thermometer thinking_face
```
Comment (flbulgarelli) [inline lib/mumukit/content_type/md_emoji.rb:119]: This is weird: `thinking_face` is not actually rendered by github (:thinking_face:), but `thinking` (:thinking:) is. 

Moreover, `thinking` is not present in this set. 

This is just an example, there may be more inconsistencies.

**Review by flbulgarelli — CHANGES_REQUESTED** [review]

**Comment by flbulgarelli** [comment]
Also, an extra question: is there a full open-source emojis set we can use?

**Comment by flbulgarelli** [comment]
Please check `wget https://api.github.com/emojis`

**Comment by flbulgarelli** [comment]
Related to https://github.com/mumuki/mumuki-bibliotheca/issues/80

**Comment by fedescarpa** [comment]
Necesitamos borrar md-emoji porque restringe los emojis y actualizarlos es problematico.

**Pull request closed without merging**